app/code/core/Mage/Adminhtml/Block/Cache/Grid.php

```diff
@@ -145,12 +127,11 @@ public function decorateStatus($value, $row, $column, $isExport)
     /**
      * Get row edit url
      *
-     * @return string
+     * @return false
```

Comment:
how does other grids behave? do they return string? or string or false? is there an interface?

Reply:
Other return string, for being clickable.